[Buildfix] Fix enable_webrtc=false build.

BUG=667527
TEST=ninja content/renderer

Review-Url: https://codereview.chromium.org/2522663003
Cr-Commit-Position: refs/heads/master@{#433753}

Author: mbjorge <mbjorge@chromium.org>	Tue Nov 22 02:39:22 2016
--- a/content/renderer/pepper/pepper_media_device_manager.cc
+++ b/content/renderer/pepper/pepper_media_device_manager.cc
@@ -89,9 +89,9 @@ void PepperMediaDeviceManager::Enumerate
                  callback, ToMediaDeviceType(type)));
 #else
   base::ThreadTaskRunnerHandle::Get()->PostTask(
-      FROM_HERE,
-      base::Bind(&PepperMediaDeviceManager::DevicesEnumerated, AsWeakPtr(),
-                 callback, ToMediaDeviceType(type), MediaDeviceInfoArray()));
+      FROM_HERE, base::Bind(&PepperMediaDeviceManager::DevicesEnumerated,
+                            AsWeakPtr(), callback, ToMediaDeviceType(type),
+                            std::vector<MediaDeviceInfoArray>()));
 #endif
 }
 
